components/about/Column.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode } from 'react';
+
+import { classNames } from '@/utils/index';
+
+import { Button } from '@/components/Button';
+
+type Props = {
+  title?: string;
+  button?: boolean;
+  buttonLink?: string;
+  buttonText?: string;
+  children: ReactNode;
+};
+
+const Column = ({ title, button, buttonLink, buttonText, children }: Props) => {
+  return (
+    <div className={classNames('max-w-[616px] space-y-10')}>
+      <h2 className="heading-2">{title && title}</h2>
+
+      {children}
+
+      {button && <Button text={buttonText ?? ''} link={buttonLink ?? ''} />}
+    </div>
+  );
+};
+
+export default Column;
```

Comment:
Nothing actually wrong with this, but here are some posts you might find interesting:

- https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad
- https://notes.webutvikling.org/default-exports/
- https://rajeshnaroth.medium.com/avoid-es6-default-exports-a24142978a7a